plugins/@grouparoo/clickhouse/src/lib/table-import/connection.ts

```diff
@@ -0,0 +1,26 @@
+import { buildConnection } from "@grouparoo/app-templates/dist/source/table";
+import { getSampleRows } from "@grouparoo/mysql/dist/lib/table-import/getSampleRows";
+import { getTables } from "@grouparoo/mysql/dist/lib/table-import/getTables";
+
+import { getColumns } from "./getColumns";
+import { getChangedRows } from "./getChangedRows";
+import { getPropertyValue } from "./getPropertyValue";
+import { getPropertyValues } from "./getPropertyValues";
+import { getChangedRowCount } from "./getChangedRowCount";
+
+export function getConnection() {
+  return buildConnection({
+    app: "clickhouse",
```

Comment:
Once you rebase from `main`, this will become 

```suggestion
    apps: ["clickhouse"],
```